layers/meta-balena-5x-owa/recipes-core/images/balena-image.inc

```diff
@@ -38,13 +38,17 @@ IMAGE_INSTALL:append = " \
                         linux-firmware-owasys \
                         wpa-supplicant \
                         udev-rules-owasys \
+                        system-maintenance \
                         system-maintenance-service \
                         rtc-tools \
                         quectel-cm \
                         switch-gsm \
                         st32-firmware \
                         chatscripts \
                         issue-owa \
+                        redis \
+                        hiredis \
+                        pollux-libs \
 "
```

Comment:
Likewise, as these were dependencies for `pollux` they are no longer required.

Reply:
Moved to [PR 28](https://github.com/balena-os/balena-owa5x/pull/28)